resource_customizations/cluster.x-k8s.io/MachinePool/health.lua

```diff
@@ -0,0 +1,51 @@
+-- Reference CRD can be found here:
+-- https://doc.crds.dev/github.com/kubernetes-sigs/cluster-api/cluster.x-k8s.io/MachinePool/v1beta1@v1.8.1
+
+function getStatusBasedOnPhase(obj, hs)
+    -- Phases can be found here: 
+    -- https://github.com/kubernetes-sigs/cluster-api/blob/release-1.8/exp/api/v1beta1/machinepool_types.go#L139-L182
+    if obj.status ~= nil and obj.status.phase ~= nil then
+        hs.message = "MachinePool is " .. obj.status.phase
+        if obj.status.phase == "Running" or
+            obj.status.phase == "Scaling" or
+            obj.status.phase == "ScalingUp" or
+            obj.status.phase == "ScalingDown" then
```

Comment:
Yeah I was not 100% sure, here when you are scaling up or down that does not mean the services is not already ready and running so that's why I put heathy. let me update.

Reply:
That's fair, some of these end up being a judgment call